docs/platform/visual/configuration.md

```diff
@@ -1,127 +1,260 @@
+from the [Analyzer](http://www.pentaho.com/product/business-visualization-analytics) application's
+visualizations menu, effectively preventing the user from creating new visualizations of this type:
 
-Examples of typical view configurations:
-- define the margins
-- configure an extension point of a CCC visualization
+```js
+var ruleSpec = {
+  select: {
+    type: "pentaho/visual/models/pie",
+    application: "pentaho-analyzer"
+  },
+  apply: {
+    isBrowsable: false
+  }
+};
+```
 
-## Configuring a visualization
+### Setting the default line width of a line chart
 
-The following is an example of a configuration module that declares two rules:
+The following rule configures the default value of the `lineWidth` property of both
+the _Line_ and the _Column/Line Combo_ stock visualizations, 
+when in the [PDI](http://www.pentaho.com/product/data-integration) application,
+to be of `2` pixels:
 
-```javascript
-define(function(){
-  
-  "use strict";
-  
-  // Fist rule: 
-  // hide pie charts
-  var noPieCharts = {
-    select: {
-      type: "pentaho/visual/models/pie"
-    },
-    apply: {
-      isBrowsable: false
+```js
+var ruleSpec = {
+  select: {
+    type: [
+      "pentaho/visual/models/line",
+      "pentaho/visual/models/barLine"
+    ],
+    application: "pentaho-det"
+  },
+  apply: {
+    props: {
+      lineWidth: {
+        value: 2
+      }
     }
-  };
-  
-  // Second rule:
-  // in Analyzer, for all Line and Bar/Line charts, 
-  // change the default line width to 2
-  // and change the default dot shape
-  var defaultLineWidthAndShapeInAnalyzer = {
-    select: {
-      application: "pentaho-analyzer",
-      type: [
-        "pentaho/visual/models/line",
-        "pentaho/visual/models/barLine"
-      ]
-    },
-    apply: {
-      props: {
-        lineWidth: {
-          value: 2
-        },
-        shape: {
-          value: "diamond"
-        }
+  }
+};
+```
+
+### Setting the default shape of points of a line chart
+
+The following rule configures the default value of the `shape` property of both
+the _Line_ and the _Column/Line Combo_ stock visualizations, 
+when in any application,
+to be the `diamond` shape:
+
+```js
+var ruleSpec = {
+  select: {
+    type: [
+      "pentaho/visual/models/line",
+      "pentaho/visual/models/barLine"
+    ]
+  },
+  apply: {
+    props: {
+      shape: {
+        value: "diamond"
+      }
+    }
+  }
+};
+```
+
+### Hiding the line width property of line charts
+
+The following rule hides the `lineWidth` property of both
+the _Line_ and the _Column/Line Combo_ stock visualizations, 
+from the Analyzer application's properties panel, 
+effectively preventing the user from changing its default value:
+
+```js
+var ruleSpec = {
+  select: {
+    type: [
+      "pentaho/visual/models/line",
+      "pentaho/visual/models/barLine"
+    ],
+    application: "pentaho-analyzer"
+  },
+  apply: {
+    props: {
+      lineWidth: {
+        isBrowsable: false
```

Comment:
How about also setting a value and describe it as "configures the default value of the `lineWidth` property and hides it from the Analyzer application's properties panel, effectively preventing the user from changing its value".

That way we show that it can be done in a _"single shot"_, no need for separate rules... The same for changing two props in a single rule.